packages/algob/sample-project/common/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_data
+.dockerignore
```

Comment:
Please also add
```
artifacts/
/algob.config-local.js
node_modules
```

Reply:
Please also add

```
.DS_Store
```
Why you add `/node_data`? Its mean `node_data` in `root` folder right? @thdailong

Reply:
I copy the .gitignore file in the same "Sample project" folder and copy to this "common" folder.
I am thinking that the node_data is in the root folder but I am not realy sure